examples/worlds/model_photo_shoot.sdf

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" ?>
+<!--
+  Ignition Gazebo Model Photo Shoot plugin demo
+
+  This will take perspective, top, front, and both sides pictures of the model:
+    ign gazebo  -s -r -v 4 --iterations 50 model_photo_shoot.sdf
```

Comment:
Unfortunately, `--` is not allowed in xml. libsdformat parses fine, but xmllint fails.
```suggestion
    ign gazebo  -s -r -v 4 \-\-iterations 50 model_photo_shoot.sdf
```